{"name": "Login con credenciales incorrectas -- @1.2 ", "status": "broken", "statusDetails": {"message": "\nYou can implement step definitions for undefined steps with these snippets:\n\n@then(u'aparece \"error\" en pantalla')\ndef step_impl(context):\n    raise NotImplementedError(u'STEP: Then aparece \"error\" en pantalla')\n\n"}, "steps": [{"name": "Given se ha inicializado el contexto", "status": "passed", "start": 1643965944007, "stop": 1643965946060}, {"name": "When se lanza el login con \"standard_user\" y \"badPass\"", "status": "passed", "start": 1643965946061, "stop": 1643965946276}, {"name": "Then aparece \"error\" en pantalla", "status": "broken", "statusDetails": {"message": "\nYou can implement step definitions for undefined steps with these snippets:\n\n@then(u'aparece \"error\" en pantalla')\ndef step_impl(context):\n    raise NotImplementedError(u'STEP: Then aparece \"error\" en pantalla')\n\n"}, "attachments": [{"name": ".text", "source": "bf88b8bf-8f62-433a-b5ef-1e10b1952da2-attachment.txt", "type": "text/plain"}], "start": 1643965946276, "stop": 1643965946277}], "parameters": [{"name": "usuario", "value": "standard_user"}, {"name": "contrasena", "value": "badPass"}, {"name": "elemento", "value": "error"}], "start": 1643965944006, "stop": 1643965946346, "uuid": "ba503d94-54ff-436f-8deb-b0278645b987", "historyId": "18c2b466ba1027e8b508183464531113", "fullName": "Pruebas sobre el login: Login con credenciales incorrectas", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Pruebas sobre el login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}